web/static/css/layout.scss

```diff
@@ -11,3 +11,9 @@
   flex: 1;
 }
 
+#main-nav {
+  a {
+    color: white;
```

Comment:
Properties should be ordered font-weight, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)